modules/@apostrophecms/piece-type/ui/apos/components/AposPiecesManagerDisplay.vue

```diff
@@ -104,8 +124,26 @@ export default {
   emits: [
     'open',
     'change',
-    'updated'
+    'updated',
+    'preview',
+    'copy',
+    'discardDraft',
+    'archive',
+    'unarchive'
   ],
+  data() {
+    const state = {
```

Comment:
keep track of hover states of rows to pass into child components

Reply:
restore